tensorflow/python/ops/math_ops.py

```diff
@@ -1467,7 +1467,8 @@ def count_nonzero(input_tensor,
   ```
 
   Args:
-    input_tensor: The tensor to reduce. Should be of numeric type, or `bool`.
+    input_tensor: The tensor to reduce. Should be of numeric type, `string`,
+      or `bool`.
```

Comment:
Can you add a note and example about the behavior of this function with dtype string to the docstring?